scripts/build_platform_config.py

```diff
@@ -286,7 +286,7 @@ def codeOutDevicePin(device, pin, definition_name):
   codeOut("#define FLASH_SAVED_CODE_START            "+str(flash_saved_code_start))
   codeOut("#define FLASH_SAVED_CODE_LENGTH           "+str(int(flash_page_size*flash_saved_code_pages)))
   codeOut("#define FLASH_MAGIC_LOCATION              (FLASH_SAVED_CODE_START + FLASH_SAVED_CODE_LENGTH - 4)")
-  codeOut("#define FLASH_MAGIC 0xDEADBEEF")
+  codeOut("#define FLASH_MAGIC 0xBADBEEF8")
```

Comment:
As mentioned in the forum, you'll end up with unreliable code when upgrading from *any build* and using previously saved code, as the code can contain pointers to functions in flash memory - so I'd rather keep this as it was before as it doesn't fix the problem at all.

For example `a=print;save()` then `a("Hello")` will most likely break *each time the firmware is updated*.

The best solution would be to change the instructions so that the saved code (or at least the magic number) was overwittten each time the firmware was updated